dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -714,13 +714,15 @@ std::vector<pingcap::coprocessor::CopTask> DAGStorageInterpreter::buildCopTasks(
         std::multimap<std::string, std::string> meta_data;
         meta_data.emplace("is_remote_read", "true");
 
-        auto tasks = pingcap::coprocessor::buildCopTasks(
+        auto tasks = pingcap::coprocessor::buildCopTasks( // Here
```

Comment:
```suggestion
        auto tasks = pingcap::coprocessor::buildCopTasks(
```
Please delete useless comments

Reply:
> Please delete useless comments

done